src/Microsoft.VisualBasic.Core/src/Microsoft.VisualBasic.Core.vbproj

```diff
@@ -28,16 +28,19 @@
     <Compile Include="Microsoft\VisualBasic\FileIO\FileSystem.Unix.vb" />
   </ItemGroup>
   <ItemGroup>
-    <Compile Include="Microsoft\VisualBasic\CallType.vb" />
-    <Compile Include="Microsoft\VisualBasic\Collection.vb" />
     <Compile Include="Microsoft\VisualBasic\ApplicationServices\StartupEventArgs.vb" />
     <Compile Include="Microsoft\VisualBasic\ApplicationServices\StartupNextInstanceEventArgs.vb" />
     <Compile Include="Microsoft\VisualBasic\ApplicationServices\UnhandledExceptionEventArgs.vb" />
+    <Compile Include="Microsoft\VisualBasic\CallType.vb" />
```

Comment:
Are we able to just use globs here?

Reply:
In CoreFX we don't use them, basically only for test assets. But my care level is low (as long as it's "*.vb" and not "*")